test/check-machines-virtualization

```diff
@@ -60,9 +60,10 @@ class TestMachinesVirtualization(machineslib.VirtualMachinesCase):
 
         def stopLibvirtSocket():
             m.execute(
-    f"systemctl stop {self.getLibvirtServiceName()}-ro.socket {self.getLibvirtServiceName()}.socket {self.getLibvirtServiceName()}-admin.socket")
+f"systemctl stop {self.getLibvirtServiceName()}-ro.socket {self.getLibvirtServiceName()}.socket {self.getLibvirtServiceName()}-admin.socket")  # noqa: E501
```

Comment:
This should also be fixed in the next round.

Reply:
I've fixed this and other comments in the current PR as there is no time limit or rush to get this fixed.